workflows/integration-testing/templates/shutdown-workflow.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: Shutdown-Workflow
+  labels:
+    workflows.argoproj.io/type: "integration-test"
+    argocd.argoproj.io/instance: integration-testing
+spec:
+  serviceAccountName: {{ .Values.serviceAccount }}
+  artifactRepositoryRef:
+    configMap: integration-test-controller-configmap
+    key: artifactRepository
+  volumes:
+  - name: testreports
+    persistentVolumeClaim:
+      claimName: {{ .Values.persistentVolume.claimName }}
+  podMetadata:
+    labels:
+      argocd.argoproj.io/instance: integration-testing
+  arguments:
+    parameters:
+    - name: date-key
+      value: "20230327"
+  entrypoint: run-tests
+  onExit: save-reports
+  templates:
+  - name: run-tests
+    dag:
+      tasks:
+      - name: shutdown
+        depends: love-stress-test
```

Comment:
Since this isn't defined in the workflow template, it's going to get upset and not work.